src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -366,6 +377,7 @@ def export_config(cmd,
         dest_kvs = __read_kv_from_config_store(dest_azconfig_client,
                                                key=SearchFilterOptions.ANY_KEY,
                                                label=dest_label if dest_label else SearchFilterOptions.EMPTY_LABEL,
+                                               tags=tags,
```

Comment:
We should not be filtering destination kvs by src tags. Source tags are only used to filter the kvs in source store that will be exported to destination.